File: java/gzip/src/main/java/com/azure/gzip/gzip/Controllers/HomeController.java
package com.azure.gzip.gzip.Controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HomeController {
    String message = "gzip-stack-examples-java-gzip";

    @GetMapping("/")
    public String index() {
        return message.repeat(60);
    }
}

File: node/gzip/controllers/homeController.js
import express from "express";

const router = express.Router();

const homeController = router.get("/", (_, res) => {
  try {
    const message = "gzip-and-brotli-stack-examples-node-gzip";
    res.send(message.repeat(50));
  } catch (error) {
    console.error(error);
  }
});

export default homeController;
